docs/gen-ai/non-normative/examples-llm-calls.md

```diff
@@ -0,0 +1,568 @@
+    I-->>-A: #U+200D
+```
+
+### GenAI client span when content capturing is disabled
+
+|   Property                      |                     Value                  |
+|---------------------------------|--------------------------------------------|
+| Span name                       | `"chat gpt-4"`                             |
+| Trace id                        | `"4bf92f3577b34da6a3ce929d0e0e4736"`       |
+| Span id                         | `"00f067aa0ba902b7"`                       |
+| `gen_ai.provider.name`          | `"openai"`                                 |
+| `gen_ai.operation.name`         | `"chat"`                                   |
+| `gen_ai.request.model`          | `"gpt-4"`                                  |
+| `gen_ai.request.max_tokens`     | `200`                                      |
+| `gen_ai.request.top_p`          | `1.0`                                      |
+| `gen_ai.response.id`            | `"chatcmpl-9J3uIL87gldCFtiIbyaOvTeYBRA3l"` |
+| `gen_ai.response.model`         | `"gpt-4-0613"`                             |
+| `gen_ai.usage.output_tokens`    | `47`                                       |
+| `gen_ai.usage.input_tokens`     | `52`                                       |
+| `gen_ai.response.finish_reasons`| `["stop"]`                                 |
+
+### GenAI client span when content capturing is enabled on span attributes
+
+|   Property                      |                     Value                  |
+|---------------------------------|--------------------------------------------|
+| Span name                       | `"chat gpt-4"`                             |
+| Trace id                        | `"4bf92f3577b34da6a3ce929d0e0e4736"`       |
+| Span id                         | `"00f067aa0ba902b7"`                       |
+| `gen_ai.provider.name`          | `"openai"`                                 |
+| `gen_ai.operation.name`         | `"chat"`                                   |
+| `gen_ai.request.model`          | `"gpt-4"`                                  |
+| `gen_ai.request.max_tokens`     | `200`                                      |
+| `gen_ai.request.top_p`          | `1.0`                                      |
+| `gen_ai.response.id`            | `"chatcmpl-9J3uIL87gldCFtiIbyaOvTeYBRA3l"` |
+| `gen_ai.response.model`         | `"gpt-4-0613"`                             |
+| `gen_ai.usage.output_tokens`    | `47`                                       |
+| `gen_ai.usage.input_tokens`     | `52`                                       |
+| `gen_ai.response.finish_reasons`| `["stop"]`                                 |
+| `gen_ai.input.messages`         | [`gen_ai.input.messages`](#gen-ai-input-messages-simple) |
+| `gen_ai.output.messages`        | [`gen_ai.output.messages`](#gen-ai-output-messages-simple) |
+
+<span id="gen-ai-input-messages-simple">`gen_ai.input.messages` value</span>
+
+```json
+[
+  {
+    "role": "system",
```

Comment:
I think only having an example showing putting system messages in input and no example of using `gen_ai.system_instructions` may lead to people overusing the former method.

Reply:
added example https://github.com/open-telemetry/semantic-conventions/pull/2179/commits/eccd1f806e426a32c98271c3ce77585492d26de2